### PR TITLE
[fix](be) Ignore negative integers in bitmap aggregates

### PR DESCRIPTION
**Pull request by HappenLee**
### What problem does this PR solve?

Problem Summary:

`bitmap_agg` and `bitmap_union_int` currently pass signed negative integers into `BitmapValue`, which converts them to unsigned bitmap members. For example, `bitmap_agg(-1)` can contain `18446744073709551615` and `bitmap_union_int(-1)` returns 1, while `to_bitmap(-1)` produces an empty bitmap.

Filter negative integers before insertion in both aggregates. Cover scalar insertion, nullable and non-nullable `bitmap_agg` batches, and the integer-only branch of the shared bitmap-count aggregate. Preserve the allocation-free batch path for all-nonnegative input and use a tracked temporary buffer for filtered batches. Generic bitmap operations and the full uint64 domain of `bitmap_union_count` are unchanged.

With this change:
- `bitmap_count(bitmap_agg(-1))` and `bitmap_union_int(-1)` return 0.
- Input `[-1, 0, 1, 1, NULL]` contributes exactly the members `{0, 1}`.
- All-negative/all-NULL groups produce an empty bitmap or a count of 0.

### Release note

`bitmap_agg` and `bitmap_union_int` now ignore negative integers, consistently with `to_bitmap`. In particular, `bitmap_union_int` counts distinct nonnegative integers rather than all distinct integers. Queries containing negative inputs can return smaller counts after upgrading.

### Check List (For Author)

- Test: add BE unit tests covering TINYINT/SMALLINT/INT/BIGINT, nullable/non-nullable input, negative minima, positive maxima, zero, duplicates, all-negative/all-NULL groups, dense batches, reset, streaming serialization and column merge paths. Also protect UINT64_MAX in `bitmap_union_count`.
  - Passed: clang-format 16 check, BE build-hygiene checks and `git diff --check`.
  - Passed: compiler syntax checks of both aggregate registration/instantiation translation units and the modified test translation unit, using the generated ASAN-UT compilation settings.
  - Attempted: `run-be-ut.sh -j48 --run --filter='BitmapIntegerAggregateTest.*:AggBitmapTest.*'`. The build stops in unchanged `common/cpp/aws_common.cpp` because the local thirdparty installation lacks `aws/core/auth/GeneralHTTPCredentialsProvider.h`; tests have not executed locally.
  - Attempted: repository clang-tidy script. Analysis is blocked by an existing unmatched NOLINTEND in `core/types.h` and existing warnings in the touched files outside the new logic. No clean clang-tidy run is claimed.
- Behavior changed: Yes, negative integers no longer contribute bitmap members or counts. NULL handling, result types, function signatures and intermediate/storage formats remain unchanged.
- Does this need documentation: Yes: https://github.com/apache/doris-website/pull/4154 updates the English/Chinese development-version documentation and clarifies the changed `bitmap_union_int` semantics.

### Review notes

The existing aggregate test helper exercises the batch, streaming and merge paths without adding a new framework. Grouped additions use the same scalar `add` entry point. No new locks, configuration, persistence changes or FE/BE fields are introduced. Filtering is applied only to integer inputs; already-materialized bitmaps are preserved. During a rolling upgrade, old BEs can still contribute old partial states, so uniform negative-input semantics require all executing BEs to contain this fix.

### CODE REVIEW

**Comment by hello-stephen** [comment]
Thank you for your contribution to Apache Doris.
Don't know what should be done next? See [How to process your PR](https://doris.apache.org/community/how-to-contribute/review-and-ci).

Please clearly describe your PR:
1. What problem was fixed (it's best to include specific error reporting information). How it was fixed.
2. Which behaviors were modified. What was the previous behavior, what is it now, why was it modified, and what possible impacts might there be.
3. What features were added. Why was this function added?
4. Which code was refactored and why was this part of the code refactored?
5. Which functions were optimized and what is the difference before and after the optimization?

**Comment by HappenLee** [comment]
run buildall

**Comment by hello-stephen** [comment]
<details>
<summary>TPC-H: <b>Total hot run time: 27572 ms</b></summary>

```
machine: 'aliyun_ecs.c7a.8xlarge_32C64G'
scripts: https://github.com/apache/doris/tree/master/tools/tpch-tools
Tpch sf100 test result on commit 1ac4fa4b5766f360033e3df7d439b58a85605e59, data reload: false

------ Round 1 ----------------------------------
============================================
q1	17709	3866	3782	3782
q2	2214	330	280	280
q3	10101	1345	771	771
q4	4680	472	352	352
q5	7487	815	558	558
q6	174	164	136	136
q7	756	778	589	589
q8	9328	1581	1563	1563
q9	5362	4149	4164	4149
q10	6767	1599	1337	1337
q11	423	268	245	245
q12	641	414	291	291
q13	18103	2612	2040	2040
q14	264	259	236	236
q15	q16	733	714	667	667
q17	1738	1201	1096	1096
q18	6427	5562	5509	5509
q19	1168	1156	977	977
q20	477	389	271	271
q21	5424	2759	2429	2429
q22	439	364	294	294
Total cold run time: 100415 ms
Total hot run time: 27572 ms

----- Round 2, with runtime_filter_mode=off -----
============================================
q1	4529	4456	4762	4456
q2	759	595	554	554
q3	4817	5293	4665	4665
q4	2226	2306	1446	1446
q5	4504	4459	4421	4421
q6	235	176	135	135
q7	1886	1778	1526	1526
q8	2255	1996	2058	1996
q9	7441	7130	6848	6848
q10	4214	4163	3762	3762
q11	510	380	346	346
q12	707	710	511	511
q13	2297	2596	1985	1985
q14	264	282	247	247
q15	q16	664	680	593	593
q17	7283	6729	6656	6656
q18	11865	11063	11846	11063
q19	1115	1004	971	971
q20	2218	2194	1898	1898
q21	5302	4412	4583	4412
q22	517	450	384	384
Total cold run time: 65608 ms
Total hot run time: 58875 ms
```
</details>

**Comment by hello-stephen** [comment]
<details>
<summary>TPC-DS: <b>Total hot run time: 154073 ms</b></summary>

```
machine: 'aliyun_ecs.c7a.8xlarge_32C64G'
scripts: https://github.com/apache/doris/tree/master/tools/tpcds-tools
TPC-DS sf100 test result on commit 1ac4fa4b5766f360033e3df7d439b58a85605e59, data reload: false

query5	4317	588	464	464
query6	433	184	175	175
query7	4809	549	300	300
query8	320	173	160	160
query9	8775	3980	3997	3980
query10	457	313	261	261
query11	5848	2182	2055	2055
query12	160	98	100	98
query13	1256	541	400	400
query14	6550	4539	4221	4221
query14_1	4022	3982	4016	3982
query15	200	199	174	174
query16	960	463	427	427
query17	920	679	537	537
query18	2463	468	332	332
query19	197	180	142	142
query20	105	103	102	102
query21	221	136	112	112
query22	12989	12982	12918	12918
query23	15438	14523	14198	14198
query23_1	14233	14048	14118	14048
query24	7790	1690	1232	1232
query24_1	1240	1233	1212	1212
query25	545	430	377	377
query26	1264	308	161	161
query27	2672	514	331	331
query28	4555	1958	1949	1949
query29	1051	565	477	477
query30	310	231	195	195
query31	880	752	632	632
query32	137	96	90	90
query33	498	305	243	243
query34	1191	1101	615	615
query35	722	755	650	650
query36	794	773	710	710
query37	149	101	88	88
query38	1849	1763	1748	1748
query39	706	670	648	648
query39_1	649	639	650	639
query40	218	117	99	99
query41	70	66	65	65
query42	92	88	91	88
query43	337	345	305	305
query44	1350	682	692	682
query45	180	179	175	175
query46	1062	1127	716	716
query47	1488	1487	1377	1377
query48	370	394	283	283
query49	602	398	295	295
query50	1007	336	247	247
query51	10580	10650	10384	10384
query52	81	85	71	71
query53	242	252	174	174
query54	254	202	192	192
query55	80	76	72	72
query56	247	232	207	207
query57	1386	1403	1422	1403
query58	232	209	219	209
query59	1966	2100	1799	1799
query60	270	236	242	236
query61	144	145	135	135
query62	399	317	265	265
query63	211	175	169	169
query64	2783	999	811	811
query65	4007	3955	3955	3955
query66	1809	433	306	306
query67	20121	20043	19899	19899
query68	3514	1596	1010	1010
query69	404	300	265	265
query70	949	901	880	880
query71	300	233	213	213
query72	2853	2481	2190	2190
query73	818	816	449	449
query74	4647	4501	4277	4277
query75	2278	2261	1904	1904
query76	2315	1081	708	708
query77	359	400	310	310
query78	8929	9154	8452	8452
query79	1290	1141	742	742
query80	536	494	383	383
query81	468	278	246	246
query82	269	159	130	130
query83	276	271	248	248
query84	308	152	124	124
query85	854	515	439	439
query86	296	246	235	235
query87	2028	1946	1824	1824
query88	3634	2767	2774	2767
query89	325	287	242	242
query90	2151	178	182	178
query91	166	152	122	122
query92	104	81	84	81
query93	1420	1451	828	828
query94	499	349	302	302
query95	699	455	332	332
query96	1014	780	336	336
query97	2471	2437	2303	2303
query98	201	189	185	185
query99	738	721	615	615
Total cold run time: 240453 ms
Total hot run time: 154073 ms
```
</details>

**Comment by hello-stephen** [comment]
<details>
<summary>ClickBench: <b>Total hot run time: 23.98 s</b></summary>

```
machine: 'aliyun_ecs.c7a.8xlarge_32C64G'
scripts: https://github.com/apache/doris/tree/master/tools/clickbench-tools
ClickBench test result on commit 1ac4fa4b5766f360033e3df7d439b58a85605e59, data reload: false

query1	0.01	0.00	0.01
query2	0.09	0.05	0.05
query3	0.26	0.14	0.13
query4	1.60	0.14	0.14
query5	0.24	0.22	0.22
query6	1.17	0.93	0.92
query7	0.04	0.01	0.01
query8	0.05	0.04	0.03
query9	0.39	0.34	0.34
query10	0.55	0.55	0.54
query11	0.20	0.14	0.14
query12	0.18	0.16	0.14
query13	0.46	0.47	0.47
query14	0.95	0.93	0.93
query15	0.62	0.60	0.58
query16	0.31	0.34	0.32
query17	1.12	1.07	1.11
query18	0.22	0.18	0.20
query19	1.98	1.92	1.96
query20	0.02	0.01	0.01
query21	15.45	0.22	0.15
query22	4.83	0.06	0.05
query23	16.17	0.31	0.13
query24	2.97	0.41	0.34
query25	0.10	0.06	0.04
query26	0.73	0.22	0.14
query27	0.05	0.04	0.04
query28	3.55	0.82	0.35
query29	12.48	4.05	3.22
query30	0.29	0.16	0.16
query31	2.77	0.56	0.31
query32	3.22	0.58	0.49
query33	3.11	3.21	3.15
query34	15.59	3.92	3.26
query35	3.25	3.23	3.24
query36	0.55	0.44	0.43
query37	0.09	0.07	0.06
query38	0.06	0.03	0.03
query39	0.04	0.03	0.03
query40	0.17	0.15	0.15
query41	0.09	0.03	0.02
query42	0.04	0.03	0.03
query43	0.05	0.03	0.03
Total cold run time: 96.11 s
Total hot run time: 23.98 s
```
</details>

**Comment by hello-stephen** [comment]
# BE UT Coverage Report
Increment line coverage `100.00% (35/35)` :tada:

[Increment coverage report](http://coverage.selectdb-in.cc/coverage/1ac4fa4b5766f360033e3df7d439b58a85605e59_1ac4fa4b5766f360033e3df7d439b58a85605e59/increment_report/index.html)
[Complete coverage report](http://coverage.selectdb-in.cc/coverage/1ac4fa4b5766f360033e3df7d439b58a85605e59_1ac4fa4b5766f360033e3df7d439b58a85605e59/report/index.html)
| Category          | Coverage           |
|-------------------|--------------------|
| Function Coverage | 63.61% (29598/46530) |
| Line Coverage     | 48.30% (308108/637936) |
| Region Coverage   | 43.91% (248682/566359) |
| Branch Coverage   | 45.45% (115560/254247) |

File: be/src/exprs/aggregate/aggregate_function_bitmap_agg.h [hunk 1]
```diff
@@ -27,6 +27,7 @@
 
 #include "core/assert_cast.h"
 #include "core/data_type/data_type_bitmap.h"
+#include "core/pod_array.h"
 #include "core/value/bitmap_value.h"
```
[the remark this answers is not in the document]
Comment (HappenLee) [reply be/src/exprs/aggregate/aggregate_function_bitmap_agg.h:31]: AGG_STATE is an experimental feature, and backward compatibility for persisted AGG_STATE data is not a requirement for this PR. Please treat that compatibility requirement as out of scope when re-evaluating this change.

**Comment by HappenLee** [comment]
/review

AGG_STATE is an experimental feature. Backward compatibility for persisted AGG_STATE data is explicitly out of scope for this PR. Please re-evaluate the persisted-state review concern under this scope and review the remaining changes normally.

**Comment by HappenLee** [comment]
/review

Please review the updated head fcb26e2e742a1d77cbb61e856b98cbe9de2f5460, especially the P2 scratch-allocation fix from https://github.com/apache/doris/pull/68219#discussion_r4052727433.

Removed both eager `values.reserve(batch_size)` calls. The existing tracked PaddedPODArray now grows only when a non-NULL, nonnegative row survives, so all-NULL/all-negative batches allocate no scratch buffer. The direct all-nonnegative `add_many` fast path is unchanged. Dense filtered input uses the container's existing geometric growth; no extra filtering pass was added.

The previously agreed scope remains unchanged: backward compatibility for persisted experimental AGG_STATE data is out of scope. This follow-up addresses P2 only; the separate mixed-version rolling-upgrade concern is not changed by these two deletions.

Validation: clang-format 16, check-format, BE header hygiene, git diff --check and syntax-only compilation of the bitmap_agg registration translation unit passed. Existing unit tests cover all-NULL, all-negative, mixed-sign and nonnegative inputs for all four signed integer widths. Local unit tests were not rerun because the previous ASAN UT build is blocked by the missing AWS GeneralHTTPCredentialsProvider.h dependency. clang-tidy was attempted and remains blocked by pre-existing diagnostics, including an unmatched NOLINTEND in core/types.h.

**Comment by hello-stephen** [comment]
# BE Regression && UT Coverage Report
Increment line coverage `100.00% (33/33)` :tada:

[Increment coverage report](http://coverage.selectdb-in.cc/coverage/68219_1ac4fa4b5766f360033e3df7d439b58a85605e59_merge/increment_report/index.html)
[Complete coverage report](http://coverage.selectdb-in.cc/coverage/68219_1ac4fa4b5766f360033e3df7d439b58a85605e59_merge/report/index.html)
| Category          | Coverage           |
|-------------------|--------------------|
| Function Coverage | 76.27% (34363/45055) |
| Line Coverage     | 61.13% (385550/630656) |
| Region Coverage   | 57.58% (324812/564077) |
| Branch Coverage   | 58.36% (147922/253458) |

**Comment by HappenLee** [comment]
run buildall

**Review by mrhhsg — APPROVED** [review]

**Comment by hello-stephen** [comment]
<details>
<summary>TPC-H: <b>Total hot run time: 27549 ms</b></summary>

```
machine: 'aliyun_ecs.c7a.8xlarge_32C64G'
scripts: https://github.com/apache/doris/tree/master/tools/tpch-tools
Tpch sf100 test result on commit fcb26e2e742a1d77cbb61e856b98cbe9de2f5460, data reload: false

------ Round 1 ----------------------------------
============================================
q1	17596	3855	3827	3827
q2	2253	340	286	286
q3	10061	1393	786	786
q4	4685	475	348	348
q5	7477	821	554	554
q6	177	166	136	136
q7	735	766	580	580
q8	9303	1525	1545	1525
q9	5362	4148	4144	4144
q10	6878	1590	1333	1333
q11	437	268	245	245
q12	632	404	294	294
q13	18097	2599	2007	2007
q14	252	262	234	234
q15	q16	732	717	654	654
q17	1832	1165	996	996
q18	6499	5598	5508	5508
q19	1154	1205	924	924
q20	489	390	262	262
q21	5614	2802	2601	2601
q22	428	357	305	305
Total cold run time: 100693 ms
Total hot run time: 27549 ms

----- Round 2, with runtime_filter_mode=off -----
============================================
q1	4521	4415	4498	4415
q2	776	590	559	559
q3	4778	5140	4721	4721
q4	2215	2295	1483	1483
q5	4593	4456	4422	4422
q6	227	179	125	125
q7	1851	1761	1497	1497
q8	2265	1988	1966	1966
q9	7459	7190	6830	6830
q10	4219	4213	3799	3799
q11	533	375	336	336
q12	689	714	498	498
q13	2274	2621	1996	1996
q14	260	269	253	253
q15	q16	668	682	595	595
q17	7269	6713	6688	6688
q18	11811	11081	11750	11081
q19	1087	993	1001	993
q20	2198	2179	1912	1912
q21	5299	4455	4571	4455
q22	510	468	393	393
Total cold run time: 65502 ms
Total hot run time: 59017 ms
```
</details>

**Comment by hello-stephen** [comment]
<details>
<summary>TPC-DS: <b>Total hot run time: 153997 ms</b></summary>

```
machine: 'aliyun_ecs.c7a.8xlarge_32C64G'
scripts: https://github.com/apache/doris/tree/master/tools/tpcds-tools
TPC-DS sf100 test result on commit fcb26e2e742a1d77cbb61e856b98cbe9de2f5460, data reload: false

query5	4332	605	493	493
query6	451	188	174	174
query7	4815	544	288	288
query8	326	183	198	183
query9	8810	3989	3982	3982
query10	455	311	261	261
query11	5784	2168	1965	1965
query12	155	103	99	99
query13	1272	573	420	420
query14	6507	4614	4264	4264
query14_1	4018	4000	3994	3994
query15	205	198	179	179
query16	1007	464	441	441
query17	935	699	537	537
query18	2443	473	338	338
query19	203	182	147	147
query20	106	104	109	104
query21	215	146	116	116
query22	13160	13042	12817	12817
query23	15507	14623	13997	13997
query23_1	13992	14205	14280	14205
query24	7716	1691	1190	1190
query24_1	1249	1247	1226	1226
query25	545	447	372	372
query26	1289	310	167	167
query27	2690	523	342	342
query28	4566	1997	1979	1979
query29	1060	580	489	489
query30	318	240	196	196
query31	896	754	674	674
query32	142	90	91	90
query33	496	302	232	232
query34	1206	1106	641	641
query35	713	778	637	637
query36	779	800	694	694
query37	147	104	93	93
query38	1832	1778	1732	1732
query39	686	667	646	646
query39_1	648	654	646	646
query40	225	126	99	99
query41	66	67	67	67
query42	95	88	92	88
query43	333	343	299	299
query44	1425	692	677	677
query45	185	178	170	170
query46	1033	1149	704	704
query47	1505	1497	1412	1412
query48	395	386	297	297
query49	589	402	284	284
query50	984	335	240	240
query51	10465	10526	10496	10496
query52	84	86	73	73
query53	244	247	177	177
query54	267	222	180	180
query55	80	74	67	67
query56	236	268	227	227
query57	1444	1405	1212	1212
query58	248	214	210	210
query59	1967	2066	1798	1798
query60	275	246	226	226
query61	147	144	147	144
query62	405	317	267	267
query63	215	174	173	173
query64	2832	980	813	813
query65	4005	3985	3933	3933
query66	1794	451	305	305
query67	19970	20339	20468	20339
query68	3151	1478	876	876
query69	410	298	265	265
query70	961	905	886	886
query71	296	232	217	217
query72	2860	2466	2175	2175
query73	848	846	407	407
query74	4704	4461	4311	4311
query75	2400	2287	1943	1943
query76	2343	1105	726	726
query77	362	393	292	292
query78	9203	9001	8482	8482
query79	1210	1161	707	707
query80	555	466	380	380
query81	460	292	241	241
query82	445	164	127	127
query83	361	267	242	242
query84	332	139	108	108
query85	820	467	376	376
query86	325	240	234	234
query87	1995	1976	1826	1826
query88	3702	2732	2683	2683
query89	346	289	243	243
query90	1897	174	181	174
query91	168	155	125	125
query92	102	80	95	80
query93	1519	1519	830	830
query94	520	305	282	282
query95	680	475	331	331
query96	1076	778	341	341
query97	2472	2455	2305	2305
query98	204	192	181	181
query99	742	716	610	610
Total cold run time: 240739 ms
Total hot run time: 153997 ms
```
</details>

**Comment by hello-stephen** [comment]
<details>
<summary>ClickBench: <b>Total hot run time: 23.88 s</b></summary>

```
machine: 'aliyun_ecs.c7a.8xlarge_32C64G'
scripts: https://github.com/apache/doris/tree/master/tools/clickbench-tools
ClickBench test result on commit fcb26e2e742a1d77cbb61e856b98cbe9de2f5460, data reload: false

query1	0.01	0.01	0.00
query2	0.10	0.05	0.05
query3	0.25	0.14	0.13
query4	1.61	0.14	0.14
query5	0.26	0.22	0.22
query6	1.16	0.97	0.91
query7	0.04	0.00	0.00
query8	0.06	0.04	0.03
query9	0.39	0.34	0.34
query10	0.57	0.55	0.56
query11	0.20	0.14	0.15
query12	0.18	0.15	0.15
query13	0.46	0.46	0.47
query14	0.96	0.94	0.94
query15	0.60	0.58	0.60
query16	0.33	0.32	0.33
query17	1.10	1.11	1.05
query18	0.22	0.20	0.20
query19	2.03	1.91	1.86
query20	0.02	0.02	0.01
query21	15.43	0.25	0.14
query22	4.77	0.05	0.05
query23	16.14	0.31	0.12
query24	3.00	0.40	0.31
query25	0.11	0.05	0.05
query26	0.74	0.21	0.16
query27	0.04	0.04	0.03
query28	3.47	0.82	0.36
query29	12.48	4.06	3.23
query30	0.28	0.15	0.17
query31	2.78	0.56	0.32
query32	3.25	0.60	0.48
query33	3.23	3.15	3.27
query34	15.52	3.93	3.29
query35	3.24	3.19	3.24
query36	0.54	0.44	0.42
query37	0.09	0.06	0.07
query38	0.05	0.04	0.03
query39	0.04	0.03	0.03
query40	0.18	0.14	0.14
query41	0.08	0.03	0.03
query42	0.03	0.04	0.03
query43	0.04	0.04	0.03
Total cold run time: 96.08 s
Total hot run time: 23.88 s
```
</details>

**Comment by hello-stephen** [comment]
# BE UT Coverage Report
Increment line coverage `100.00% (33/33)` :tada:

[Increment coverage report](http://coverage.selectdb-in.cc/coverage/fcb26e2e742a1d77cbb61e856b98cbe9de2f5460_fcb26e2e742a1d77cbb61e856b98cbe9de2f5460/increment_report/index.html)
[Complete coverage report](http://coverage.selectdb-in.cc/coverage/fcb26e2e742a1d77cbb61e856b98cbe9de2f5460_fcb26e2e742a1d77cbb61e856b98cbe9de2f5460/report/index.html)
| Category          | Coverage           |
|-------------------|--------------------|
| Function Coverage | 63.66% (29616/46525) |
| Line Coverage     | 48.34% (308324/637874) |
| Region Coverage   | 43.93% (248812/566331) |
| Branch Coverage   | 45.47% (115603/254227) |

**Comment by hello-stephen** [comment]
# BE Regression && UT Coverage Report
Increment line coverage `100.00% (33/33)` :tada:

[Increment coverage report](http://coverage.selectdb-in.cc/coverage/68219_fcb26e2e742a1d77cbb61e856b98cbe9de2f5460_merge/increment_report/index.html)
[Complete coverage report](http://coverage.selectdb-in.cc/coverage/68219_fcb26e2e742a1d77cbb61e856b98cbe9de2f5460_merge/report/index.html)
| Category          | Coverage           |
|-------------------|--------------------|
| Function Coverage | 76.29% (34368/45050) |
| Line Coverage     | 61.17% (385744/630594) |
| Region Coverage   | 57.60% (324902/564049) |
| Branch Coverage   | 58.38% (147966/253438) |

**Comment by hello-stephen** [comment]
# BE Regression && UT Coverage Report
Increment line coverage `100.00% (33/33)` :tada:

[Increment coverage report](http://coverage.selectdb-in.cc/coverage/68219_fcb26e2e742a1d77cbb61e856b98cbe9de2f5460_merge/increment_report/index.html)
[Complete coverage report](http://coverage.selectdb-in.cc/coverage/68219_fcb26e2e742a1d77cbb61e856b98cbe9de2f5460_merge/report/index.html)
| Category          | Coverage           |
|-------------------|--------------------|
| Function Coverage | 76.29% (34368/45050) |
| Line Coverage     | 61.17% (385744/630594) |
| Region Coverage   | 57.60% (324902/564049) |
| Branch Coverage   | 58.38% (147966/253438) |

**Comment by hello-stephen** [comment]
# BE Regression && UT Coverage Report
Increment line coverage `100.00% (33/33)` :tada:

[Increment coverage report](http://coverage.selectdb-in.cc/coverage/68219_fcb26e2e742a1d77cbb61e856b98cbe9de2f5460_merge/increment_report/index.html)
[Complete coverage report](http://coverage.selectdb-in.cc/coverage/68219_fcb26e2e742a1d77cbb61e856b98cbe9de2f5460_merge/report/index.html)
| Category          | Coverage           |
|-------------------|--------------------|
| Function Coverage | 76.30% (34375/45050) |
| Line Coverage     | 61.20% (385925/630594) |
| Region Coverage   | 57.65% (325162/564049) |
| Branch Coverage   | 58.42% (148061/253438) |

**Comment by hello-stephen** [comment]
# BE Regression && UT Coverage Report
Increment line coverage `100.00% (33/33)` :tada:

[Increment coverage report](http://coverage.selectdb-in.cc/coverage/68219_fcb26e2e742a1d77cbb61e856b98cbe9de2f5460_merge/increment_report/index.html)
[Complete coverage report](http://coverage.selectdb-in.cc/coverage/68219_fcb26e2e742a1d77cbb61e856b98cbe9de2f5460_merge/report/index.html)
| Category          | Coverage           |
|-------------------|--------------------|
| Function Coverage | 76.29% (34368/45050) |
| Line Coverage     | 61.17% (385743/630594) |
| Region Coverage   | 57.60% (324893/564049) |
| Branch Coverage   | 58.38% (147965/253438) |
